shell/ptty_windows.go

```diff
@@ -11,6 +13,25 @@ import (
 	"github.com/gruntwork-io/terragrunt/options"
 )
 
+func PrepareConsole() {
+	enableVirtualTerminalProcessing(os.Stderr)
+	enableVirtualTerminalProcessing(os.Stdout)
+}
+
+func enableVirtualTerminalProcessing(file *os.File) {
```

Comment:
Please add a comment explaining what this method does, including those links to StackOverflow / GitHub

Reply:
Added to `PrepareConsole`